Screenbox/Package.appxmanifest

```diff
@@ -218,7 +218,7 @@
         </uap:Extension>
         <uap:Extension Category="windows.fileTypeAssociation">
           <uap:FileTypeAssociation Name="quicktime">
-            <uap:DisplayName>ms-resource:QuickTimeFileFriendlyName</uap:DisplayName>
+            <uap:DisplayName>mms-resource:ManifestResources/FileDisplayNameQuickTime</uap:DisplayName>
```

Comment:
I think you made a typo here. `mms-resource`

Reply:
Fixed